### ISSUE TITLE
spam in monitor logs and driver logs

### ISSUE THREAD

**Issue by AmeerHajAli** [ray-project/ray#13457]
my code:
```
In [5]: @ray.remote(num_cpus=2) 
   ...: def f(): 
   ...:     time.sleep(5) 
   ...:                                                                                                                                                                    

In [6]: result = ray.get([f.remote() for _ in range(150)]) 
```
My cluster yaml is the regular example-full.yaml with max_workers = 100.

issue 1:
The autoscaler logs prints (why 10091+?):
`{'CPU': 2.0}: 10091+ pending tasks/actors`

issue 2:
and prints (I think this should be debug):
`2021-01-14 16:58:29,121	INFO resource_demand_scheduler.py:164 -- Placement group demands: []`

issue 3:
I still see:
```
2021-01-14 17:31:34,595	INFO load_metrics.py:102 -- LoadMetrics: Removed mapping: 172.31.28.16 - 1610645494.3798053
2021-01-14 17:31:34,596	INFO load_metrics.py:102 -- LoadMetrics: Removed mapping: 172.31.31.99 - 1610645494.3797066
2021-01-14 17:31:34,596	INFO load_metrics.py:102 -- LoadMetrics: Removed mapping: 172.31.29.51 - 1610645494.3798382
2021-01-14 17:31:34,596	INFO load_metrics.py:102 -- LoadMetrics: Removed mapping: 172.31.17.146 - 1610645494.3797739
2021-01-14 17:31:34,596	INFO load_metrics.py:109 -- LoadMetrics: Removed 4 stale ip mappings: {'172.31.28.16', '172.31.31.99', '172.31.29.51', '172.31.17.146'} not in {'172.31.27.239', '172.31.30.15'}
```
issue 4:
The driver prints some weird spam:
```
2021-01-14 16:19:23,127 WARNING worker.py:1039 -- The node with node id 2b9379ed02041fc65b060d85392fe899100b6ce774391fd1e88ce3d7 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:19:23,128	WARNING worker.py:1039 -- The node with node id 46396e1d00c0c46d0f74ee91e2353101bc9bd4865fe1fd17a627d32c has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:19:23,230	WARNING worker.py:1039 -- The node with node id 5fed491a3225c5455bc0123c3fe44484c4619acfda834b2d554386c4 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:19:23,230	WARNING worker.py:1039 -- The node with node id 79ffc3ea64a349856756d70eac3eb5cc5b628b262775c971d29e5d1c has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:19:59,566	WARNING worker.py:1039 -- The node with node id 5e9b19bcc972d7d01a833f2e877537643a50520f6e61e93f7b772612 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:20:49,712	WARNING worker.py:1039 -- The node with node id 57caf75e4c45921d2efd006cb06c0355e1bbb31b85682f9deef19ac0 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:20:49,804	WARNING worker.py:1039 -- The node with node id 22eff1d189fb8eadebda7875a2c6ce65080da2bd04cd609af08a3f02 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:20:49,805	WARNING worker.py:1039 -- The node with node id 69470d5269f714fa75e7dbefbdf01a47123570ab8fe36b62255d5c6a has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:20:49,805	WARNING worker.py:1039 -- The node with node id ed728791bd0753a0805a939da7a7ddfc8e7ad3c233f157cd897e2e37 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:20:49,906	WARNING worker.py:1039 -- The node with node id d660bc696124bf8662c93f25c24d8abf309c67dce5ebc34da6f9c8d3 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:21:25,836	WARNING worker.py:1039 -- The node with node id 496512ef133f056aeae73e370d27c5c966d175781ca2651e62549e04 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:21:25,938	WARNING worker.py:1039 -- The node with node id 5c9b8d8341d2171b148adc3611e3dd5574adc9c93489afe9cd14c455 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:21:37,355	WARNING worker.py:1039 -- The node with node id b9ae1ef2a4da1c38d39531e394cd095c54b11c14f3006928eba3d5fe has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
2021-01-14 16:21:37,457	WARNING worker.py:1039 -- The node with node id 3356a2d3d80a896046495445caa7769a9258283725bfd0b0ade521b7 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a raylet crashes unexpectedly or has lagging heartbeats.
```

- [ ] I have verified my script runs in a clean environment and reproduces the issue.
- [ ] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/master/installation.html).

**Comment by wuisawesome** [comment]
Issue 1: This seems like a real issue. We should try to understand if this is a core issue, or an autoscaler issue. 

Issue 2: Let's fix it quickly, it should be trivial. 

Issue 3 and 4: Are these being constantly repeated? If not, this is intended behavior. If you're unhappy with these messages then you should file a separate _feature_ request to have the autoscaler gracefully call `ray stop` before terminating a node. 

Also, in general, we should probably make separate github issues to track orthogonal problems.

**Comment by wuisawesome** [comment]
It looks like issue 1 here is a duplicate of #13431

**Comment by AmeerHajAli** [comment]
@wuisawesome , re issue 3+4, yes, they were constantly repeated

**Comment by AmeerHajAli** [comment]
Should be closed after #13431.
